cim-forward-route/src/main/java/com/crossoverjie/cim/route/controller/RouteController.java

```diff
@@ -131,10 +131,12 @@ public BaseResponse<NULLBody> p2pRoute(@RequestBody P2PReqVO p2pRequest) throws
     public BaseResponse<NULLBody> offLine(@RequestBody ChatReqVO groupReqVO) {
         BaseResponse<NULLBody> res = new BaseResponse();
 
-        CIMUserInfo cimUserInfo = userInfoCacheService.loadUserInfoByUserId(groupReqVO.getUserId());
+        if(userInfoCacheService.CheckUserLoginStatus(groupReqVO.getUserId())){
```

Comment:
Why not directly modify the return value of [loadUserInfoByUserId](https://github.com/crossoverJie/cim/blob/44085441b3baa060bf61f222db84e102314d1ba5/cim-forward-route/src/main/java/com/crossoverjie/cim/route/service/UserInfoCacheService.java#L22) to `Optional<CIMUserInfo>`, and then call the `offLine` function in `ifPresent`?

Reply:
Absolutely, It is a good idea that it will be midified right now.